addons/ballistics/CfgMagazineWells.hpp

```diff
@@ -0,0 +1,186 @@
+class CfgMagazineWells {
+
+    class CBA_65x39_MX {
+        ADDON[] = {
+            "ACE_30Rnd_65x39_caseless_mag_Tracer_Dim",
+            "ACE_30Rnd_65x39_caseless_green_mag_Tracer_Dim"
+        };
+    };
+    class CBA_65x39_MX_XL {
+        ADDON[] = {
+            "ACE_100Rnd_65x39_caseless_mag_Tracer_Dim",
+            "ACE_200Rnd_65x39_cased_Box_Tracer_Dim"
```

Comment:
I think this is a 200 rnd belt and probably not right for mx
was used in Stoner 99 / LMG_Mk200_F

Reply:
yah, the 200rd is a belt for the Mk200.
I will make a PR to cba to add a magwell for it.

Reply:
https://github.com/CBATeam/CBA_A3/pull/1007

Reply:
moved to that well
and it won't cause any problems to use the well config before cba is updated